pom.xml

```diff
@@ -376,7 +375,7 @@
                         <exclude>**/src/test/resources/**</exclude>
                         <exclude>**/target/**</exclude>
                     </excludes>
-                    <lineEnding>LF</lineEnding>
+                    <lineEnding>AUTO</lineEnding>
```

Comment:
Ow, nvm them...

But with move from LF to AUTO if all is LF since quite a while?